spec/requests/static_pages_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe "StaticPages", type: :request do
+  describe "#home" do
+    it "should have the title Home" do
+      visit '/static_pages/home'
+
+      expect(page).to have_title('Expenses | Home')
+    end
+
+    it "should have basic info about the app" do
+      visit '/static_pages/home'
+
+      expect(page).to have_content('Tracking expenses made easy')
+    end
+  end
+
+  describe "#about" do
+    it "should have the title About" do
+      visit '/static_pages/about'
+
+      expect(page).to have_title('Expenses | About')
+    end
+
+    it "should have the content 'About'" do
+      visit '/static_pages/about'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.